searx/engines/hex.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""hex.pm"""
+
+from urllib.parse import urlencode
+from dateutil import parser
+
+
+about = {
+    "website": "https://hex.pm/",
+    "wikidata_id": None,
+    "official_api_documentation": "https://github.com/hexpm/hexpm/blob/main/lib/hexpm_web/controllers/api/package_controller.ex",
+    "use_official_api": True,
+    "require_api_key": False,
+    "results": "JSON",
+}
+
+categories = ["it", "packages"]
+
+
+# engine dependent config
+paging = True
+search_api = "https://hex.pm/api/packages/?"
+
+
+def request(query: str, params):
+    args = urlencode({"page": params["pageno"], "search": query})
+    params["url"] = search_api + args
+    return params
+
+
+def response(resp):
+    results = []
+    content = resp.json()
+    for package in content:
```

Comment:
```suggestion
    for package in resp.json():
```
nit: this can be inlined